rustlang/install.sh

```diff
@@ -2,9 +2,35 @@
 set -e
 set -u
 
+# shellcheck disable=SC2016
+
 function __install_rust() {
+
     # Straight from https://rustup.rs/
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+    # Remove the default env file in the default shell config
+    if [ -z "$(command -v sd)" ]; then
+        webi sd
+    fi
+
+    echo ""
+    echo 'Removing from shell config '. "$HOME/.cargo/env"''
```

Comment:
delete all of this stuff. None of this is necessary or useful.

Since `$HOME/.cargo/env` is only for bash, not for fish, it doesn't do any good to edit or remove it.